integration-tests/avro/pom.xml

```
@@ -21,13 +21,12 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.apache.camel.quarkus</groupId>
-        <artifactId>camel-quarkus-build-parent-it</artifactId>
+        <artifactId>camel-quarkus-integration-tests</artifactId>
         <version>1.1.0-SNAPSHOT</version>
-        <relativePath>../../../poms/build-parent-it/pom.xml</relativePath>
     </parent>
 
-    <artifactId>camel-quarkus-avro-integration-test</artifactId>
-    <name>Camel Quarkus :: Avro :: Integration Test</name>
+    <artifactId>camel-quarkus-integration-test-avro</artifactId>
+    <name>Camel Quarkus :: Integration Test :: Avro</name>
```

Minor nit - for consistency with other itest modules, change to:
```suggestion
    <name>Camel Quarkus :: Integration Tests :: Avro</name>
```